openghg/store/_boundary_conditions.py

```diff
@@ -56,6 +60,7 @@ def read_file(
         species: str,
         bc_input: str,
         domain: str,
+        source_format: str = "boundary_conditions",
```

Comment:
As per comment within `_boundary_conditions.py` this should be "openghg"
```suggestion
        source_format: str = "openghg",
```